uni/lib/controller/local_storage/database/app_database.dart

```diff
@@ -32,12 +41,28 @@ class AppDatabase {
   /// The version of this database.
   final int version;
 
+  /// Getter to determine if the session is persistent.
+  Future<bool> get persistentSession async {
+    _persistentSession ??=
+        await PreferencesController.getPersistentUserInfo() != null;
+    return _persistentSession!;
+  }
+
   /// Returns an instance of this database.
   Future<Database> getDatabase() async {
     _db ??= await initializeDatabase();
     return _db!;
   }
 
+  Future<void> saveToDatabase(T data);
```

Comment:
Can this be `protected` to prevent external usage?

Reply:
Dart doesn't have a `protected` access modifier.